packages/helm/symphony/files/symphony-api.json

```diff
@@ -212,7 +212,7 @@
             "type": "managers.symphony.jobs",
             "properties": {
               "providers.state": "mem-state",
-              "baseUrl": "http://symphony-service:8080/v1alpha2/",
+              "baseUrl": "https://symphony-service:8081/v1alpha2/",
```

Comment:
If the port is configured by symphony.apiContainerPortHttp, it should be used here as well, right?